lib/Versions

```diff
@@ -29,6 +30,7 @@ LIBNXZ_0.61.0 {
     nx_crc32;
     nx_crc32_combine;
     nx_crc32_combine64;
+    crc32_ppc;
```

Comment:
Why is this a global symbol?

Reply:
To allow users to use the crc32 optimized version if ZLIB_API is not defined.